common/djangoapps/student/models.py

```diff
@@ -1403,6 +1408,9 @@ class LinkedInAddToProfileConfiguration(ConfigurationModel):
         "honor": ugettext_lazy(u"{platform_name} Honor Code Certificate for {course_name}"),
         "verified": ugettext_lazy(u"{platform_name} Verified Certificate for {course_name}"),
         "professional": ugettext_lazy(u"{platform_name} Professional Certificate for {course_name}"),
+        CourseMode.NO_ID_PROFESSIONAL_MODES[0]: ugettext_lazy(
```

Comment:
I'm really not comfortable with this. Please be explicit and use the mode's name as the key in this dictionary.